manageiq-gems-pending.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "fog-openstack",           "~> 0.3"
   s.add_runtime_dependency "handsoap",                "=  0.2.5.5"
   s.add_runtime_dependency "linux_admin",             "~> 2.0"
-  s.add_runtime_dependency "manageiq-password",       "~> 0.3"
+  s.add_runtime_dependency "manageiq-password",       "~> 1.0"
```

Comment:
Can this be loosened to `< 2`?

Reply:
Is that not effectively the same thing?

Reply:
Oh i see - that would also allow the 0.3 in this particular repo - interesting idea.

Reply:
Yeah, this doesn't appear to depend on anything that broke in the major version change, so it would ease the upgrade process